recipes/libastra/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set version = "1.9.9" %}
+{% set build = 0 %}
+
+package:
+  name: libastra
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomopy/astra-toolbox/archive/refs/tags/{{ version }}.tar.gz
+  sha256: fc8c1bc2553cd93e570e1991b44df301c89ce7d9521033672f2d9d421af94beb
+
+build:
+  number: {{ build }}
+  string: "{{ "cuda" + cuda_compiler_version|string }}h{{ PKG_HASH }}_{{ build }}"
+  skip: true  # [(not linux) or cuda_compiler_version == "None"]
+  script:
+    cmake
+    -G Ninja
+    -DCMAKE_BUILD_TYPE=Release
+    ${CMAKE_ARGS} ${SRC_DIR} && ninja install
+
+requirements:
+  build:
+    - {{ compiler('cuda') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost-cpp
+    - cudatoolkit
+
+test:
+  requires:
+    - boost-cpp
+  source_files:
+    - tests/astratest
+  commands:
```

Can you also check for the existence of `libastra${SHLIB_EXT}` here?
```suggestion
  commands:
    - test -f $PREFIX/lib/libastra${SHLIB_EXT}
```

---

```suggestion
  commands:
    - test -f $PREFIX/lib/libastra${SHLIB_EXT}.{{ version }}
```